indexer-api/src/infra/api.rs

```diff
@@ -300,6 +300,37 @@ where
     }
 }
 
+pub(crate) trait UnshieldedAddressResultExt<T> {
```

Comment:
Please move trait and impl to the `v1` module, because this is for `v1::UnshieldedAddressFormatError`.